java/arcs/core/storage/DirectStore.kt

```diff
@@ -406,6 +411,11 @@ class DirectStore<Data : CrdtData, Op : CrdtOperation, T> /* internal */ constru
     }
 
     companion object {
+        /** The write-back thread pool is shared among all [DirectStore]s. */
+        private val writeBackThreads = Executors.newCachedThreadPool {
+            Thread(it).apply { name = "WriteBack #$id" }
+        }
```

Comment:
This will not work for `arcs.core`, since `Excecutors` is part of the JDK. I think we'll need a way to inject this type of thing from the platform-specific code somewhere.

Reply:
Will figure out a way of moving out of arcs.core.
Maybe like what we do for log.

Reply:
Moved to StorageService.kt